app/views/meeting.status.html

```diff
@@ -34,9 +36,10 @@ <h4 class="db-margin-10-bottom">{{msc.uiName}}</h4>
             </div>
             <h4 ng-if='!msc.loading' class="list-group-item ng-class:[msc.mtgStatusClass(msc.meeting)]" translate translate-cloak>{{msc.stringId(msc.meeting)}}</h4>
 
-            <div class="list-group-item ng-class:[{'db-opt-bg db-font-clr-white': (msc.isSelected(topic) && !msc.isMobile)}]" style="overflow: hidden;"
-                ng-repeat="topic in msc.meeting.topicList track by $index">
-                <div class="db-flex" db-confirm confirm-enabled="msc.hasUnsavedData && !msc.isSelected(topic)" confirm-config="msc.unsavedConfig" ng-click="msc.topicSelected(topic)">
+            <div class="list-group-item ng-class:[{'db-opt-bg db-font-clr-white': (msc.isSelected(topic) && !msc.isMobile), 'db-item-disabled': !msc.canAccess(topic)}]"
```

Comment:
The class handles only visualization, shouldn't this be replace using a ng-disabled to actually disable the click action? Or if that doesn't work add a condition to ng-click.